### PR TITLE
Bold parent page header

### PR DESCRIPTION
**Pull request by mohit-codecombat**
Made bolder H1, H3, and H4.

Live-coding-classes-from-CodeCombat-4
Live-coding-classes-from-CodeCombat-3

### CODE REVIEW

File: app/views/landing-pages/parents/PageParents.vue [hunk 1]
```diff
@@ -1261,3 +1261,9 @@ export default {
 }
 
 </style>
+
+<style>
```
Comment (AndrewJakubowicz) [inline app/views/landing-pages/parents/PageParents.vue:1265]: Two small changes.

1. Could we put this in the scoped style tag block ( `<style scoped>`) above?
Unless we need to for something external to the component (i.e. tooltips), this leaves us at risk of the style leaking across the single page app.

2. Nitpick: isn't the `id h1` specificity enough to override other h1 rules? Is there a reason `!important` is needed?

**Review by AndrewJakubowicz — CHANGES_REQUESTED** [review]
Great PR! Small nitpick on global CSS. We can leave it if the style is needed on other pages.
Also prefer we avoid `!important` unless it's required.

Thanks for the screenshots! 😄

**Comment by mohit-codecombat** [comment]
Removed `!important` from css style

File: app/views/landing-pages/parents/PageParents.vue (hunk 1 above)
Comment (mohit-codecombat) [reply app/views/landing-pages/parents/PageParents.vue]: Added css inside `Scoped` style and put same css inside `PageParentsJumbotron.vue` component to make h1 bolder in `/parents` page.

File: app/views/landing-pages/parents/PageParentsJumbotron.vue [hunk 2]
```diff
@@ -121,4 +121,8 @@ export default {
     }
   }
 
+  #parent-page h1, #parent-page h4 {
+    font-weight: 700 !important;
```
Comment (AndrewJakubowicz) [inline app/views/landing-pages/parents/PageParentsJumbotron.vue:125]: From the scoped style, you no longer need to reference the id `#parent-page`, as it doesn't appear in the html template.

You could scope it within the `.top-jumbotron` class? I.e. `.top-jumbotron h1`. You can remove the `h4` selector as there isn't an h4 in this vue component.

Finally we don't need the `!important`.

File: app/views/landing-pages/parents/PageParents.vue [hunk 3]
```diff
@@ -1260,4 +1260,8 @@ export default {
   border-radius: 20px;
 }
 
+#parent-page h1, #parent-page h4 {
+  font-weight: 700 !important;
```
Comment (AndrewJakubowicz) [inline app/views/landing-pages/parents/PageParents.vue:1264]: Required `!important`?

**Review by AndrewJakubowicz — CHANGES_REQUESTED** [review]
Maybe the commit didn't push, but the `!important` parts are still there.

File: app/views/landing-pages/parents/PageParents.vue (hunk 3 above)
Comment (mohit-codecombat) [reply app/views/landing-pages/parents/PageParents.vue]: No, I removed `!important`

File: app/views/landing-pages/parents/PageParentsJumbotron.vue (hunk 2 above)
Comment (mohit-codecombat) [reply app/views/landing-pages/parents/PageParentsJumbotron.vue:125]: Updated accordingly

**Review by AndrewJakubowicz — CHANGES_REQUESTED** [review]
@mohit-codecombat , just checking that you can see what I see through diff with master.

Attached is a screenshot.
Screen Shot 2020-12-17 at 1 46 02 PM

Let me know if you're not able to repro! Thanks!

File: app/views/landing-pages/parents/PageParentsJumbotron.vue (hunk 2 above)
Comment (mohit-codecombat) [reply app/views/landing-pages/parents/PageParentsJumbotron.vue]: Updated css changes in `PageParentsJumbotron.vue`

**Comment by mohit-codecombat** [comment]
It my bad that I missed this. I swapped css and removed `!important`.

**Review by AndrewJakubowicz — APPROVED** [review]
Awesome! LGTM!

**Pull request merged**